### ISSUE TITLE
Add tg:loaded attribute to trams

### ISSUE THREAD

**Issue by kuroda** [oiax/tgweb#146]
When an element in a tram has a `tg:loaded` attribute, its value is added to the element's class attribute when the `window.load` event occurs.

**Comment by kuroda** [comment]
As another idea, what about the idea of introducing a Scheduler?

```html
<div 
  tg:scheduler 
  class="transition duration-500" 
  tg:init="opacity-0" 
  tg:0="opacity-100" 
  tg:1000="bg-red-500"
>
  Fade In
</div>
```

Initially, the value of the `class` attribute of this `<div>` element is `"transition duration-500 opacity-0"`.
The moment the page loads, its value changes to `"transition duration-500 opacity-100"`.
Then, 1000 milliseconds later, its value changes to `"transition duration-500 bg-red-500"`.

**Comment by kuroda** [comment]
Closed by 9792cbf
